src/index.js

```diff
@@ -1 +1,112 @@
-import _ from 'lodash';
+// import _ from 'lodash';
```

Comment:
Kindly remove commented code before merging.


It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))